vars/RedHat.yml

```diff
@@ -14,4 +14,7 @@ __crayfish_packages:
   - tesseract-langpack-srp
   - ffmpeg
   - poppler-utils
+  - python-psycopg2
+  - python-mysqldb
```

Comment:
Should be `MySQL-python` instead of `python-mysqldb` and `mysql` instead of `mysql-client` for CENTOS.